.vscode/launch.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
In general, is it not useful to have VS Code assets for repos (to help in running/debugging)?

Reply:
I think that the way we do it in the project is by means of [`.devcontainer`](https://containers.dev/) instead of directly adding .vscode configuration directly. That way you address the issue of both helping the developer to have a sane VS Code configuration to run but also stablish a sane/baseline development environment.

For comparison, the .NET SDK use https://github.com/dapr/dotnet-sdk/blob/master/.editorconfig instead. Supposedly IDE independent. The python SDK used .devcontainer solution -- https://github.com/dapr/python-sdk/blob/master/.devcontainer/devcontainer.json#L7-L20

Reply:
I think my question is a bit different; I view VS Code task/launch configurations as being independent of the existence of a Dev container definition.  The former simplifies launching the application (or samples or tests) in the debugger, in whatever environment VS Code *happens* to be running and the latter ensures an known-good environment in which VS Code *can* be run.  There is *some* overlap (in terms of both allowing recommendation/installation of specific extensions as well as settings), but a Dev container definition would still require task/launch configurations for debugging.

Anyway, in my opinion, both are useful to have.

Reply:
Generally speaking I don’t like to have ide-specific files being shared because they are a kind of user’s preference, but that’s only a opinion, I think we should ask for .NET SDK maintainers at least to have a consensus whether this should be used or not. Would you mind to confirm that with Hal @philliphoff ?

Reply:
Will ask Hal, but removed for the time being.

Reply:
Everyone seems to agree that it's not desirable to have these in the repo, so I've put my VS Code bias aside and removed them.